rules/integrations/okta/credential_access_user_impersonation_access.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/08/19"
+maturity = "production"
+updated_date = "2021/07/20"
+integration = "okta"
+
+[rule]
+author = ["Elastic"]
+description = """
+A user has initiated a session impersonation granting them access to our environment with the permissions of the user
+they are impersonating. This would likely indicate Okta administrative access and should only ever occur if requested
+and expected.
+"""
+from = "now-30m"
+index = ["filebeat-*", "logs-okta*"]
+interval = "15m"
+language = "kuery"
+license = "Elastic License v2"
+name = "Okta User Session Impersonation"
+note = """## Config
+
+The Okta Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule."""
+references = [
+    ""
```

Comment:
needs references

Reply:
Maybe https://twitter.com/1_0_1_1_1/status/1506191124452417540, this was the one referenced on slack

Reply:
Perfect 👍 . Gonna used the archived version, since twitter and all

Reply:
```suggestion
    "https://web.archive.org/web/20220322084856/https://twitter.com/1_0_1_1_1/status/1506191124452417540"
```